core-plugin/src/com/google/gct/idea/debugger/CloudDebugProcess.java

```diff
@@ -501,8 +501,17 @@ public SaveAndExitAction() {
 
     @Override
     public void actionPerformed(AnActionEvent event) {
-      ExitDialog exitDialog = new ExitDialog(getXDebugSession().getProject());
```

Comment:
I don't think so, it does a better job with the buttons though, maybe I should change the the dialog here to reflect those button names.